tide/core/PixelStreamInteractionDelegate.cpp

```diff
@@ -83,19 +83,28 @@ void PixelStreamInteractionDelegate::doubleTap( const QPointF position )
     emit notify( deflectEvent );
 }
 
-void PixelStreamInteractionDelegate::tapAndHold( const QPointF position )
+void PixelStreamInteractionDelegate::tapAndHold( const QPointF position,
+                                                 const uint numPoints )
 {
     deflect::Event deflectEvent = _getNormEvent( position );
     deflectEvent.type = deflect::Event::EVT_TAP_AND_HOLD;
+    deflectEvent.key = numPoints;
```

Comment:
Yes, redesigning the event is becoming more and more needed :-) but it will take some effort...